src/core/lib/security/credentials/external/pluggable_auth_external_account_credentials.cc

```diff
@@ -242,19 +256,26 @@ void PluggableAuthExternalAccountCredentials::RetrieveSubjectToken(
   // expired.
   if (output_file_path_ != "") {
     SliceWrapper content_slice;
-    grpc_error_handle error =
-        grpc_load_file(output_file_path_.c_str(), 0, &content_slice.slice);
+    error = grpc_load_file(output_file_path_.c_str(), 0, &content_slice.slice);
     if (error.ok()) {
-      absl::string_view output_file_content =
-          StringViewFromSlice(content_slice.slice);
-      CreateExecutableResponse(std::string(output_file_content));
-      // If the cached output file has an executable response that was
-      // successful and un-expired, return the subject token.
-      if (executable_response_.success &&
-          !isExpired(executable_response_.expiration_time)) {
-        FinishRetrieveSubjectToken(executable_response_.subject_token,
-                                   absl::OkStatus());
-        return;
+      std::string output_file_content =
+          std::string(StringViewFromSlice(content_slice.slice));
+      // If the output_file is blank, call the executable
+      if (output_file_content != "") {
+        executable_response_ =
+            CreateExecutableResponse(std::string(output_file_content), &error);
+        if (executable_response_ == nullptr) {
+          FinishRetrieveSubjectToken("", error);
```

Comment:
Is there a way to nest the error here? i.e. have a top level error that has the message "Could not read executable response from the specified output file" and then the inner error says the actual error that happened during parsing

Reply:
I am not sure about a way to nest errors.